libraries/chain/util/sps_processor.cpp

```diff
@@ -296,6 +296,26 @@ void sps_processor::run( const block_notification& note )
 {
    remove_old_proposals( note );
    make_payments( note );
+   record_funding( note );
```

Comment:
I think this should be done before making payments so that the inflation is recorded in account history before it is paid out to proposals.